app/js/lib/foxbox/foxbox.js

```diff
@@ -164,15 +164,14 @@ export default class Foxbox extends Service {
           boxes
             .filter((box) => box.timestamp - now >= 0)
             .map((box) => {
-                // NOTE(sgiles): There is consideration to allow
-                // only "local_origin" and "tunnel_origin", removing the
-                // need to parse message - this merges the relevant message
-                // fields into the main object
-                const { local_origin, tunnel_origin } =
-                  JSON.parse(box.message);
-                box.local_origin = local_origin;
-                box.tunnel_origin = tunnel_origin;
-                return Object.freeze(box);
+              // NOTE(sgiles): There is consideration to allow
+              // only "local_origin" and "tunnel_origin", removing the
+              // need to parse message - this merges the relevant message
+              // fields into the main object
+              const { local_origin, tunnel_origin } = JSON.parse(box.message);
+              const client = box.client;
+
+              return Object.freeze({ local_origin, tunnel_origin, client });
```

Comment:
__note:__ not really an issue, but now we have ```box.client``` and ```foxbox.clientId``` that are essentially the same thing... But I'm fine if you feel it's OK.